hazelcast/proxy/vector_collection.py

```diff
@@ -106,6 +111,18 @@ def delete(self, key: Any) -> Future[None]:
         check_not_none(key, "key can't be None")
         return self._delete_internal(key)
 
+    def optimize(self, index_name: str = None) -> Future[None]:
+        request = vector_collection_optimize_codec.encode_request(self.name, index_name)
+        return self._invoke(request)
+
+    def clear(self) -> Future[None]:
+        request = vector_collection_clear_codec.encode_request(self.name)
+        return self._invoke(request)
+
+    def size(self) -> Future[int]:
```

Comment:
Could you add API docs to these methods?

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/679/commits/a76fc95fb840fe6b5338097d0fefbabaccf514b5